core/build.sbt

```diff
@@ -67,8 +69,6 @@ addCompilerPlugin(dependencies.macroparadise.plugin)
 
 initialCommands in console := """
 import nelson._
```

Comment:
I fixed this in my PR :-) https://github.com/getnelson/nelson/pull/92/files#diff-10e6c0c6d9998a5d5b6e19e45f1485abR71